sla/sla_agent.py

```diff
@@ -76,13 +76,8 @@ def validate_nodes(self, nodes):
             self.logger.info(f'The nodes to be monitored : {nodes}')
 
         for node in nodes:
-            host = GOOD_IP if self.is_test_mode else node['ip']
             if os.system("ping -c 1 " + GOOD_IP + " > /dev/null") == 0:
```

Comment:
I thought we're using `pingparsing` here, not the system calls

Reply:
yep, we're using pingparsing for getting metrics... here was a quick system call just before a node pinging for checking if it's all OK with network (pinging 8.8.8.8 - for example)... anyway - changed for pingparsing too :)